src/classes/CRLP_RollupProcessor.cls

```diff
@@ -560,10 +560,26 @@ public inherited sharing class CRLP_RollupProcessor {
      * @return Boolean
      */
     private Boolean needsSkewModeOverrideSet(SObject updatedRecord, Boolean needsUpdate) {
-        if (((Decimal) updatedRecord.get('npo02__NumberOfClosedOpps__c') >= maxRelatedOppsForNonLDVMode ||
-            (Decimal) updatedRecord.get('npo02__NumberOfMembershipOpps__c') >= maxRelatedOppsForNonLDVMode) ||
-            (updatedRecord.getSObjectType() == Contact.SObjectType) &&
-            (Decimal) updatedRecord.get('Number_of_Soft_Credits__c') >= maxRelatedOppsForNonLDVMode)
+        Boolean addSkewField = false;
+
+        if (updatedRecord.get('npo02__NumberOfClosedOpps__c') != null &&
+            (Decimal) updatedRecord.get('npo02__NumberOfClosedOpps__c') >= maxRelatedOppsForNonSkewMode)
+        {
+            addSkewField = true;
+
+        } else if (updatedRecord.get('npo02__NumberOfMembershipOpps__c') != null &&
+            (Decimal) updatedRecord.get('npo02__NumberOfMembershipOpps__c') >= maxRelatedOppsForNonSkewMode)
+        {
+            addSkewField = true;
+
+        } else if (updatedRecord.getSObjectType() == Contact.SObjectType &&
+            updatedRecord.get('npo02__NumberOfMembershipOpps__c') != null &&
+            (Decimal) updatedRecord.get('npo02__NumberOfMembershipOpps__c') >= maxRelatedOppsForNonSkewMode)
+        {
```

Comment:
Sorry, I was curious if this was still an issue or not. Just tested the following in ApexAnon with a null value in the field and the typecast did not throw an exception:
```
Contact c = [SELECT npo02__NumberOfMembershipOpps__c FROM Contact LIMIT 1];
System.debug((Decimal)c.get('npo02__NumberOfMembershipOpps__c'));
```

Reply:
@force2b You were very clear that you were unsure.  I should have tested rather than treating your comment as a change suggestion.  My bad.  Reverted to the previous version.